digital_land_frontend/render.py

```diff
@@ -67,6 +65,15 @@ def by_organisation(self, rows):
 
         return result
 
+    def add_row_to_org_dict(self, organisation, row, organisation_dict):
```

Comment:
maybe just change `by_organisation` to `self.by_organisation` and you can get rid of the last parameter.